src/main/java/com/xpandit/plugins/xrayjenkins/model/ServerConfiguration.java

```diff
@@ -96,7 +96,7 @@ public FormValidation doCheckCredentialId(
         }
             
         if (StringUtils.isBlank(value)) {
-            return FormValidation.warning("Not filling the Credentials is not required, but you will have to fill the in each Build Task.");
+            return FormValidation.warning("Filling the Credentials is not required. They will have to be filled in each Build task.");
```

Comment:
Should this message be more clear? From what I understand you can only fill in credentials for the Jira instance on a build task if none are selected in the "Configure System" screen.